Movies.Web/Controllers/UserController.cs

```diff
@@ -1,36 +1,38 @@
-﻿using Microsoft.AspNetCore.Authorization;
+﻿using AutoMapper;
+using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc;
 using Movies.BL.Managers;
 using Movies.BL.Services;
 using Movies.Web.Managers;
+using Movies.Web.ViewModel.User;
 
 namespace Movies.Web.Controllers
 {
     [Authorize]
     public class UserController : BaseController
     {
-        private readonly UsersManager _usersManager;
         private readonly IUserManager _userManager;
-        public UserController(UsersManager usersManager,AuthenticationManager authenticationManager, IUserManager userManager) : base (authenticationManager)
+        private readonly IMapper _mapper;
+        public UserController(AuthenticationManager authenticationManager, IUserManager userManager, IMapper mapper) : base (authenticationManager)
         {
-            _usersManager = usersManager;
             _userManager = userManager;
+            _mapper = mapper;
         }
         public IActionResult Index()
         {
-            var loggedUser = _usersManager.MapUser(User);
+            var loggedUser = _mapper.Map<UserViewModel>(User);
             return View(loggedUser);
         }
-        public ActionResult AddMovie(int id)
+        public ActionResult AddMovie(int movieId)
         {
             var user = _userManager.GetUserByEmail(User.Email);
```

Comment:
Why? :)
You have the user in User property and go to the db the get the same user again by its email.